seed/challenges/intermediate-bonfires.json

```diff
@@ -19,11 +19,11 @@
         "sumAll([1, 4]);"
       ],
       "tests": [
-        "expect(sumAll([1, 4])).to.be.a('Number');",
-        "expect(sumAll([1, 4])).to.equal(10);",
-        "expect(sumAll([4, 1])).to.equal(10);",
-        "expect(sumAll([5, 10])).to.equal(45);",
-        "expect(sumAll([10, 5])).to.equal(45);"
+        "assert(typeof(sumAll([1, 4])) === \"number\", 'The result should be a number');",
```

Comment:
This could use [`assert.isNumber`](http://chaijs.com/api/assert/#isNumber) instead.